app/controllers/turbo_devise_controller.rb

```diff
@@ -2,9 +2,9 @@ class TurboDeviseController < ApplicationController
   class Responder < ActionController::Responder
     def to_turbo_stream
       controller.render(options.merge(formats: :html))
-    rescue ActionView::MissingTemplate => error
+    rescue ActionView::MissingTemplate => e
```

Comment:
Have to admit I like the more explicit `error` but can live with `e` as is a common usage.